container/lxd/manager.go

```diff
@@ -287,11 +287,12 @@ func (m *containerManager) networkDevicesFromConfig(netConfig *container.Network
 	if len(netConfig.Interfaces) > 0 {
 		return DevicesFromInterfaceInfo(netConfig.Interfaces, machineID)
 	} else if netConfig.Device != "" {
+		// For the eth0 case do not generate an interface name, instead let
```

Comment:
... and remove this from here, which in the absence of context may be confusing.